src/Fixer/Comment/HeaderCommentFixer.php

```diff
@@ -112,7 +112,14 @@ public function getDefinition(): FixerDefinitionInterface
 
     public function isCandidate(Tokens $tokens): bool
     {
-        return $tokens->isMonolithicPhp();
+        if ($tokens->isMonolithicPhp()) {
+            $openTagIndex = $tokens[0]->isGivenKind(T_INLINE_HTML) && $tokens->count() > 1 ? 1 : 0;
+            if (!$tokens[$openTagIndex]->isGivenKind(T_OPEN_TAG_WITH_ECHO)) {
+                return true;
+            }
+        }
+
+        return false;
```

Comment:
In each fixer modified in this PR it can be just:

```suggestion
        return $tokens->isMonolithicPhp() && $tokens->isTokenKindFound(T_OPEN_TAG);
```

Since `isMonolithicPhp()` already checks if there is open tag **OR** short echo tag, we can just narrow it to `T_OPEN_TAG` and it works as expected.

Reply:
Very helpful review 👍